grouper/permissions.py

```diff
@@ -193,31 +188,6 @@ def grant_permission_to_service_account(session, account, permission, argument="
     session.commit()
 
 
-def disable_permission(session, permission_name, actor_user_id):
-    """Set a permission as disabled.
-
-    Args:
-        session(models.base.session.Session): database session
-        permission_name(str): name of permission in question
-        actor_user_id(int): id of user who is disabling the permission
-    """
-    if permission_name in (entry[0] for entry in SYSTEM_PERMISSIONS):
```

Comment:
I vote we keep this check, because I really don't want to be on-call when someone accidentally disables `grouper.permission.auditor` and something crazy happens like the background processor trying to repeatedly "promote" all non-auditor owners :)

Reply:
It's still present. https://github.com/dropbox/merou/blob/master/grouper/usecases/disable_permission.py#L60  This is just removing dead code because every disable permission code path now calls the usecase instead.

Reply:
Ahh I was only looking through the immediately added code, thanks